spec/ValidationAndPasswordsReset.spec.js

```diff
@@ -527,6 +527,39 @@ describe('Custom Pages, Email Verification, Password Reset', () => {
       });
   });
 
+  it('succeeds sending a password reset email with case insensitive email', done => {
+    reconfigureServer({
+      appName: 'coolapp',
+      publicServerURL: 'http://localhost:1337/1',
+      emailAdapter: MockEmailAdapterWithOptions({
+        fromAddress: 'parse@example.com',
+        apiKey: 'k',
+        domain: 'd',
+      }),
+    })
+      .then(() => {
+        const user = new Parse.User();
+        user.setPassword('asdf');
+        user.setUsername('zxcv');
+        user.set('email', 'testCaseInsensitive@parse.com');
```

Comment:
Use @example.com

Reply:
I just used the other test cases as a template. Same for the other code review comments in `spec/ValidationAndPasswordsReset.spec.js`.

Reply:
Yes, unfortunately existing tests are not always a good template for new tests (no async/await, nested try/catch instead of awaitExpect, etc). In this case we should comply with [RFC 2606](https://tools.ietf.org/html/rfc2606) and use example.com.

Reply:
Changed.